WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -782,6 +869,25 @@ - (void)syncContentEnded
 }
 
 
+#pragma mark - Async Loading Helpers
+
+- (void)setupWithPostID:(NSNumber *)postID siteID:(NSNumber *)siteID
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    ReaderPostService *service      = [[ReaderPostService alloc] initWithManagedObjectContext:context];
+    __weak __typeof(self) weakSelf  = self;
+    
+    [service fetchPost:postID.integerValue forSite:siteID.integerValue success:^(ReaderPost *post) {
```

Comment:
It was mentioned at the WWDC (2014!) that it's okay to use dot notation for both, calculated and stored properties.
As long as the code that gets executed doesn't alter the internal state, and returns a value, (imho) it should be fine. Plus that's the Swifty way!.

However, i would definitely **not** use dot notation to call something named... say... 'setGravatarWithImageURL` (since that would be triggering a network request, and thus, an async process).

I've tried to spot the exact session, but couldn't find it. I think it was in one of the first sessions!.